frontend/lib/api/form/pollingstation/PollingStationFormController.tsx

```diff
@@ -255,7 +261,7 @@ export function PollingStationFormController({
     acceptWarnings = false,
     aborting = false,
     continueToNextSection = true,
-  }: SubmitCurrentFormOptions = {}) => {
+  }: SubmitCurrentFormOptions = {}): Promise<ApiResult<SaveDataEntryResponse>> => {
```

Comment:
```suggestion
  }: SubmitCurrentFormOptions = {}): Promise<ApiResult<SaveDataEntryResponse, ApiError>> => {
```
to enforce that only `ApiError` errors can be returned